README.md

```diff
@@ -53,15 +50,15 @@
 
 ### Deprecated
 
-| Terra Package      | Version | Status | Dependencies |
-|--------------------|---------|--------|--------------|
-| terra-clinical-action-header | [![NPM version](https://badgen.net/npm/v/terra-clinical-action-header)](https://www.npmjs.org/package/terra-clinical-action-header) | ![Deprecated](https://badgen.net/badge/status/Deprecated/grey)| ![terra-clinical-action-header](https://badgen.net/badge/dependencies/n%2Fa/grey) |
-| terra-clinical-app-delegate |[![NPM version](https://badgen.net/npm/v/terra-clinical-app-delegate)](https://www.npmjs.org/package/terra-clinical-app-delegate) | ![Deprecated](https://badgen.net/badge/status/Deprecated/grey)| ![terra-clinical-app-delegate](https://badgen.net/badge/dependencies/n%2Fa/grey) |
-| terra-clinical-error-view | [![NPM version](https://badgen.net/npm/v/terra-clinical-error-view)](https://www.npmjs.org/package/terra-clinical-error-view) | ![Deprecated](https://badgen.net/badge/status/Deprecated/grey)| ![terra-clinical-error-view](https://badgen.net/badge/dependencies/n%2Fa/grey) |
-| terra-clinical-modal-manager |[![NPM version](https://badgen.net/npm/v/terra-clinical-modal-manager)](https://www.npmjs.org/package/terra-clinical-modal-manager) | ![Deprecated](https://badgen.net/badge/status/Deprecated/grey)| ![terra-clinical-modal-manager](https://badgen.net/badge/dependencies/n%2Fa/grey) |
-| terra-clinical-no-data-view | [![NPM version](https://badgen.net/npm/v/terra-clinical-no-data-view)](https://www.npmjs.org/package/terra-clinical-no-data-view) | ![Deprecated](https://badgen.net/badge/status/Deprecated/grey)| ![terra-clinical-no-data-view](https://badgen.net/badge/dependencies/n%2Fa/grey) |
-| terra-clinical-site |[![NPM version](https://badgen.net/npm/v/terra-clinical-site)](https://www.npmjs.org/package/terra-clinical-site) | ![Deprecated](https://badgen.net/badge/status/Deprecated/grey)| ![terra-clinical-site](https://badgen.net/badge/dependencies/n%2Fa/grey) |
-| terra-clinical-slide-group |[![NPM version](https://badgen.net/npm/v/terra-clinical-slide-group)](https://www.npmjs.org/package/terra-clinical-slide-group) | ![Deprecated](https://badgen.net/badge/status/Deprecated/grey)| ![terra-clinical-slide-group](https://badgen.net/badge/dependencies/n%2Fa/grey) |
```

Comment:
These were all 404s, so I removed the column. If we still want them, let me know and I'll throw them back in.